Resources/views/CRUD/edit_orm_one_to_many.html.twig

```diff
@@ -106,7 +106,7 @@ file that was distributed with this source code.
                                 </div>
 
                                 {% if nested_group_field['_delete'] is defined %}
-                                    {{ form_row(nested_group_field['_delete']) }}
+                                    {{ form_row(nested_group_field['_delete'],  { 'label': ('action_delete'|trans({}, 'SonataAdminBundle')) } ) }}
```

Comment:
> IMO we don't need a break here, because we have many indentions caused by the html structure. 

This does not change anything, because editors dont shift right when the whole part of the file in the view port is indented, do they? This is still hard to read. This is a problem, it could be solved with include statements. I guess we also could remove the extra level of indention that is here for no apparent reason, on [this line](https://github.com/core23/SonataDoctrineORMAdminBundle/blob/master/Resources/views/CRUD/edit_orm_one_to_many.html.twig#L17)

> Btw. we have many lines in this file who have much longer lines.

And many services have the whole container directly injected in them. There are many problems in this project, I don't think we should view the existing code as a reason to do the same.